Documentation/images/swift3.md

```diff
@@ -15,11 +15,12 @@ It also takes into account any namespacing folder in your Assets Catalogs (i.e.
 
 ## Customization
 
-You can customize some elements of this template by overriding the following parameters when invoking `swiftgen` in the command line, using `--param <paramName> <newValue>`
+You can customize some elements of this template by overriding the following parameters when invoking `swiftgen` in the command line, using `--param <paramName>=<newValue>`
```

Comment:
Nah, I'd make a separate PR changing those lines to point to the StencilSwiftKit docs (or new SwiftGen docs) explaining how to customise templates.